### PR TITLE
New syndicate cosmetics, turfs, and spawners.

### PR DESCRIPTION
**Pull request by WJohn**
This adds a new syndicate vest, tweaks the HoS's beret and eyepatch sprites (it covered BOTH EYES) a bit, and adds haevacht's new med hardsuit sprites from #27691 (and thereby makes that PR obsolete). 

Adds 2x2 and 3x3 engines for really big shuttles that are definitely not happening any time soon no sir. 

Also energy variants of syndicate turrets, which lack a cover and have laser cannons for lethal mode.

Lastly, adds some proper spawners for these syndies, and fixes the fact that lavaland syndies weren't ACTUALLY part of the syndicate faction.

Teaser pic of things to come:

 
 <- Taken from haevacht's PR.

I'm fully aware that this is precisely NOT how you make a PR, and it should be cut up into smaller or separated pieces.

### CODE REVIEW

**Comment by Haevacht** [comment]
Includes my CMO sprites because merge conflicts and my client were not cooperating, at all. This is is simpler, and prevents him from having conflicts with it.

**Comment by GunHog** [comment]
Yay! I got to name something! :D

**Comment by ChangelingRain** [comment]
Please do not put giant floor signs in floors.dmi it's bad practice and makes it harder to edit
Similarly, please put the giant engines in a 96x96 dmi file so they're not split up

**Comment by WJohn** [comment]
While I was able to move the plaques to their own dmi, the shuttle engines have some vision issues that come with being 2x2 and 3x3. Remie showed me how to fix the hitboxes, but the vision is still screwed.

**Comment by ChangelingRain** [comment]
add `appearance_flags = 0` somewhere in the huge engine object definition.

File: code/game/turfs/simulated/floor/plasteel_floor.dm [hunk 1]
```diff
@@ -338,6 +339,10 @@
 	name = "Commemorative Plaque"
 	desc = "\"This is a plaque in honour of our comrades on the G4407 Stations. Hopefully TG4407 model can live up to your fame and fortune.\" Scratched in beneath that is a crude image of a meteor and a spaceman. The spaceman is laughing. The meteor is exploding."
 
+/turf/open/floor/plasteel/plaque
```
Comment (ChangelingRain) [inline code/game/turfs/simulated/floor/plasteel_floor.dm]: these shouldn't be floors at all

Comment (WJohn) [inline code/game/turfs/simulated/floor/plasteel_floor.dm]: I suppose I'll try decals, then.

**Comment by Repukan** [comment]
Star fury more like star furry lmao

**Comment by WJohn** [comment]
literally was debating on whether to rename it or not because someone was going to make that exact comment. nice.

**Comment by xxalpha** [comment]
WJ WROTE CODE, DELTA ALERT

**Comment by WJohn** [comment]
>wrote

>literally copy paste from what's already there or from what other people helped every tiny step of the way

**Comment by WJohn** [comment]
oh for shit's sake.

**Comment by RemieRichards** [comment]
I am Happy about this
I am not bEing forced to do this
I am doing this of my own free wilL and volition
haPpy day.

**Comment by Steelpoint** [comment]
I have a distinct concern this may have accidently overwrote some of my marine additions.

**Comment by Haevacht** [comment]
IHIEILP.
Is this nyngenthorifoyion for "this is a great pr and perfect"?

**Comment by GunHog** [comment]
No, it means "HELP" lol

**Pull request merged**